mmv1/templates/terraform/examples/data_fusion_instance_psc.tf.erb

```diff
@@ -0,0 +1,39 @@
+resource "google_data_fusion_instance" "<%= ctx[:primary_resource_id] %>" {
+  name             = "<%= ctx[:vars]["instance_name"] %>"
+  region           = "us-central1"
+  type             = "BASIC"
+  private_instance = true
+
+  network_config {
+    connection_type = "PRIVATE_SERVICE_CONNECT_INTERFACES"
+    private_service_connect_config {
+      network_attachment      = google_compute_network_attachment.psc.id
+      unreachable_cidr_block = "192.168.0.0/25"
+    }
+  }
+
+  <%= ctx[:vars]['prober_test_run'] %>
+}
+
+resource "google_compute_network" "psc" {
+  name                    = "<%= ctx[:vars]["network_name"] %>"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "psc" {
+  name   = "<%= ctx[:vars]["subnet_name"] %>"
+  region = "us-central1"
+
+  network       = google_compute_network.psc.id
+  ip_cidr_range = "10.0.0.0/16"
+}
+
+resource "google_compute_network_attachment" "psc" {
```

Comment:
`google_compute_network_attachment` is not in `google-provider` module and https://github.com/GoogleCloudPlatform/magic-modules/pull/10966 will solve the issue.